RemoteHealthcare/VRMethod.cs

```diff
@@ -466,6 +466,29 @@ public static string GenerateRoute(ref Connection connection, List<(JArray, JArr
             return routeRespond.data.uuid;
         }
 
+        public static void AddRoad(ref Connection connection, string routeID)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)